content/en/docs/how-to/plugin-management.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Install and manage notation plugins
+description: Install and manage notation plugins
+weight: 9
+---
+
+Plugins for `notation` provide additional functionality and integration. For example, the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin allows you to sign a Notation-generated payload with a certificate in [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview?wt.mc_id=azurelearn_inproduct_oss_notaryproject).
+
+## Install a plugin
+
+To install a plugin, download the plugin binary, create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/`, and add the plugin binary to that directory. The following example downloads and installs the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin assuming the environment variables are set for the plugin version and install path for your development computer.
+
+{{% alert title="Warning" color="primary" %}}
+The following example does not work unless you set the environment variables, and other plugins may follow a different process for downloading. You also need to know the specific location `{NOTATION_LIBEXEC}/plugins/` on your development computer. For more details on the default location of that directory on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/concepts/directory-structure" >}}).
+{{% /alert %}}
+
+```console
```

Comment:
Shouldn't we tell the user what env variables they need to set? There seems to be expectation that `${tar_file}` and `${install_path}` are already set. Maybe we should expand on the example to use specific Notation version so the TAR is known and also set example install path (let's say Linux)

Reply:
+1 requesting for examples

Reply:
I see where you're coming from. I refactored the section to make it easier to follow and understand.